Makefile

```diff
@@ -6,14 +6,25 @@ build:
 generate:
 	go generate ./...
 
+install-hooks:
+	cp -r git/hooks/* .git/hooks/
+
 log:
 	tail -f *.log
 
+openapi-spec-check-updates:
+	make openapi-spec-download
+	./scripts/check-openapi-changed.sh
+
+openapi-spec-download:
+	curl -s -o ld-openapi.json https://app.launchdarkly.com/api/v2/openapi.json
+
+openapi-spec-update:
```

Comment:
If we see a failure, we can run this locally to update the OpenAPI spec file, regenerate the resource commands, manually check anything we want, and commit the changes.